packages/devtools_app/lib/src/framework/landing_screen.dart

```diff
@@ -177,7 +157,7 @@ class _ConnectDialogState extends State<ConnectDialog>
             ),
             const SizedBox(width: defaultSpacing),
             ElevatedButton(
-              onPressed: actionInProgress ? null : _connect,
+              onPressed: actionInProgress ? null : () => unawaited(_connect()),
```

Comment:
this is one of the more unfortunate cases for the lints. They make you call out the cases where you are firing and forgetting async methods. On the other hand, that does make the code look a bit ugly.

Reply:
It would be interesting if there was a version of the button class that played nicely with async callbacks. For example, the ideal version wouldn't `unawait` the future (which just drops it on the ground) and would instead make the button show as disabled until the onPressed callback completes. That would probably be generally useful in DevTools. We have hacked around it with custom logic on some pages but a general solution might be feasible.